status/Makefile

```diff
@@ -47,4 +47,5 @@ create-service-mysql:
 
 clean:
 
-.PHONY: setup setup-dev run deploy create-service-mysql setup-db migrate-db help clean
+
+.PHONY: setup setup-dev run deploy create-service-mysql setup-db migrate-db help clean check-vulnerabilities
```

Comment:
Did you need to actually add the empty target to this Makefile?

Reply:
Nope, it's been removed